app/assets/stylesheets/shelf_items.scss

```diff
@@ -0,0 +1,131 @@
+@import "variables";
+
+.shelf-items {
+  .banner.banner-primary-1 .banner-content {
+    padding: 3rem 2rem 2rem;
+
+    h4 {
+      text-align: center;
+      color: white;
+    }
+  }
+
+  .shelf {
+    flex-wrap: wrap;
+    justify-content: center;
+    height: 100%;
+
+    .shelf-item .brew-link {
+      text-align: center;
+      margin-top: 1rem;
+
+      a {
+        display: inline-block;
+        font-weight: 600;
+        font-size: 1.2rem;
+        padding: 0.5rem 1.5rem;
+        border-radius: 2rem;
+        border: 1px solid $color-primary-2;
+        color: $color-primary-2;
+        transition: background-color 0.25s, color 0.25s, border-color 0.25s;
+
+        &:hover, &:focus {
+          background-color: $color-primary-opacity-10;
+          border-color: $color-primary-opacity-10;
+          color: $color-primary-0;
+        }
+      }
+    }
+  }
+}
+
+.shelf-container {
+  h2 {
+    padding: 0 1rem;
+    display: block;
+  }
+
+  .shelf {
+    display: flex;
+
+    &.dashboard-shelf {
+      overflow: scroll;
+      justify-content: flex-start;
+    }
+
+    .shelf-item {
+      //border: 1px solid darkslateblue;
+      flex: 1 0 0;
+      padding: 2rem;
+      border-radius: 2rem;
+      margin: 1rem;
+      max-width: 350px;
+      min-width: 350px;
```

Comment:
I've tried not to use fixed width cards thus far but rather width percentages as they scale better and you get less white space. If you're happy for me to then I can apply this here?

Reply:
yep, I'm good with that.

Reply:
resolving this for now - this can be a second pass.